TarSCM/scm/base.py

```diff
@@ -270,6 +270,10 @@ def prep_tree_for_archive(self, subdir, outdir, dstname):
              os.path.samefile(os.path.dirname(src), dst)):
             return
 
+        r_path = os.path.realpath(src)
+        if self.clone_dir not in r_path:
```

Comment:
Maybe

`if not r_path.startswith(self.clone_dir)` 

would be a more complete check?